allennlp/common/params.py

```diff
@@ -72,7 +72,7 @@ def pop(self, key: str, default: Any = DEFAULT):
                 raise ConfigurationError("key \"{}\" is required at location \"{}\"".format(key, self.history))
         else:
             value = self.params.pop(key, default)
-        logger.param(self.history + key + " = " + str(value))
+        logger.param(self.history + key + " = " + str(value))  # type: ignore
```

Comment:
because `Logger` doesn't come with a `param` method, we monkey-patched it there ourselves, and mypy can't deal with monkey-patching